src/AVFoundation/AVCaptureVideoPreviewLayer.cs

```diff
@@ -0,0 +1,36 @@
+#if !WATCH && !TVOS
+
+using System;
+
+using XamCore.Foundation;
+using XamCore.CoreFoundation;
+using XamCore.ObjCRuntime;
+using XamCore.AudioToolbox;
+
+namespace XamCore.AVFoundation {
+	public partial class AVCaptureVideoPreviewLayer {
+
+		public enum InitMode {
+			WithConnection,
+			WithNoConnection,
```

Comment:
that one should be decorated with `[iOS (8,0), Mac (10,2)]`,  it's the only hint we can provide to developers for the API availability